esbuild.mjs

```diff
@@ -0,0 +1,28 @@
+import * as esbuild from "esbuild";
+
+const isWatch = process.argv.includes("--watch");
+
+/** @type {import("esbuild").BuildOptions} */
+const options = {
+    entryPoints: {
+        App: "./Resources/Private/JavaScript/index.tsx",
+        Grid: "./Resources/Private/JavaScript/layoutGrid.tsx",
+    },
+    external: ["@neos-project/utils-helpers"],
```

Comment:
this hack can be hopefully remove if no react ui compontent relies on the `isNil` helper anymore ... which i think i fixed at some point